### PR TITLE
Adjust resources

### PR DESCRIPTION
**Pull request by FreshGyan**
Reducing resource requests to match what the actual usage is like, left the limits still reasonable.

Also playing around with some pod numbers to see how that works.

### CODE REVIEW

File: pipeline/config-prod.groovy [hunk 1]
```diff
@@ -341,33 +341,33 @@ environments {
...
-                    cpu_limit = "150m"
-                    memory_request = "256Mi"
-                    memory_limit = "512Mi"
-                    replica_min = 2
-                    replica_max = 4
+                    cpu_request = "10m"
+                    cpu_limit = "50m"
+                    memory_request = "96Mi"
+                    memory_limit = "160Mi"
+                    replica_min = 3
```
Comment (pdittaro) [inline pipeline/config-prod.groovy]: Any reason for increasing the nginx min replicas?

File: pipeline/config-prod.groovy [hunk 2]
```diff
@@ -341,33 +341,33 @@ environments {
...
+                    memory_request = "512Mi"
+                    memory_limit = "2Gi"
                     uwsgi_threads = 2
                     uwsgi_processes = 4
                     replica_min = 2
                     replica_max = 4
                 }
                 python_lite {
-                    cpu_request = "100m"
+                    cpu_request = "10m"
```
Comment (pdittaro) [inline pipeline/config-prod.groovy]: This seems a little low compared to test, and doesn't have a cpu limit.

File: pipeline/config-prod.groovy (hunk 1 above)
Comment (FreshGyan) [reply pipeline/config-prod.groovy]: After openshift 101, Jason A/S recommended we try with lower resources and more pods to see how things are working. 
Just so if a node gets evacuated or is faulty or something, we don't have downtime.
Im also curious to see how well this works and if it actually performs better somehow

File: pipeline/config-prod.groovy (hunk 2 above)
Comment (FreshGyan) [reply pipeline/config-prod.groovy]: The limit is right underneath it 😅 
But yeah, the python_lite pod is actually not using more than 0.001 CPU in general, so I reduced the CPU to reflect that.

Good catch about test tho, updated test/dev to reflect the same values

**Pull request merged**